configs/windows/mozconfig

```diff
@@ -12,34 +10,46 @@ if test "$ZEN_CROSS_COMPILING"; then
 
   export CROSS_BUILD=1
   CROSS_COMPILE=1
-  TOOLS=$(echo ~)/win-cross
-
-  export WIN_UCRT_REDIST_DIR="$(echo ~)/win-cross/vs2022/Windows Kits/10/Redist/10.0.22621.0/ucrt/DLLs/x64"
-  export WIN32_REDIST_DIR="$(echo ~)/win-cross/vs2022/VC/Redist/MSVC/14.38.33135/x64/Microsoft.VC143.CRT"
 
-  export MIDL="$TOOLS/wine/bin/widl"
```

Comment:
Wine's MIDL can't build ARM64, it fails with a keyerror on the cpu type. Need to use the Windows SDK MIDL compiler (which is what's used by default when not specified).

Reply:
There's no reason to use different compilers versions by architecture since the windows sdk one works for all of them but sure it can be moved to the platform section.